buildSrc/src/test/kotlin/com/google/firebase/gradle/plugins/PublishingPluginTests.kt

```diff
@@ -245,7 +247,8 @@ class PublishingPluginTests {
       .withArguments(
         "-PprojectsToPublish=${projects.joinToString(",") { it.name }}",
         "-PreleaseName=m123",
-        "firebasePublish"
+        "firebasePublish",
       )
       .withPluginClasspath()
+  // .forwardOutput()
```

Comment:
Intended, as I wanted to leave a clue for future debugging work- but I think Javadocs denoting this would serve this purpose better. I've updated the function as such.